services/harmony/app/frontends/workflow-ui.ts

```diff
@@ -261,6 +271,12 @@ function tableQueryToJobQuery(tableQuery: TableQuery, isAdmin: boolean, user: st
       in: tableQuery.allowUsers,
     };
   }
+  if (tableQuery.providerValues.length) {
```

Comment:
nit: For consistency maybe change this to `if (tableQuery.providerValues.length > 0)` since you use that style in `data-operation.ts` below.

Reply:
Good point.